src/github/service/try_cmd.rs

```diff
@@ -0,0 +1,19 @@
+use crate::github::api::operations::merge_branches;
+use crate::github::api::RepositoryClient;
+use octocrab::models::pulls::PullRequest;
+
+pub async fn try_command(repo: &RepositoryClient, pr: PullRequest) -> anyhow::Result<()> {
+    log::debug!("Executing try on {}/{}", repo.name(), pr.id);
+
+    // Just merge the PR directly right now. Let's hope that tests pass :)
+    merge_branches(
+        repo,
+        &pr.base.ref_field,
+        &pr.head.ref_field,
+        "Bors merge commit",
```

Comment:
nit: can we already do the commit message bors uses?

> Auto merge of {number} - {author}:{branch_name}, r={reviewer}

Reply:
Ok, I added the message and basic handling of merge errors.